src/BizHawk.Client.EmuHawk/MainForm.cs

```diff
@@ -2048,20 +2048,9 @@ private void HandlePlatformMenus()
 					GBSubMenu.Visible = true;
 					SameBoyColorChooserMenuItem.Visible = Emulator is Sameboy sameboy && !sameboy.IsCGBMode(); // palette config only works in DMG mode
 					break;
-				case VSystemID.Raw.SNES when Emulator is LibsnesCore { IsSGB: true }: // doesn't use "SGB" sysID
-					SNESSubMenu.Text = "&SGB";
-					SNESSubMenu.Visible = true;
-					break;
-				case VSystemID.Raw.SNES when Emulator is LibsnesCore { IsSGB: false }:
-					SNESSubMenu.Text = "&SNES";
-					SNESSubMenu.Visible = true;
-					break;
-				case VSystemID.Raw.SNES when Emulator is BsnesCore bsnesCore:
-					SNESSubMenu.Text = bsnesCore.IsSGB ? "&SGB" : "&SNES";
-					SNESSubMenu.Visible = true;
-					break;
-				case VSystemID.Raw.SNES when Emulator is SubBsnesCore subBsnesCore:
-					SNESSubMenu.Text = subBsnesCore.IsSGB ? "&SGB" : "&SNES";
+				case VSystemID.Raw.SGB when Emulator is BsnesCore or SubBsnesCore:
+				case VSystemID.Raw.SNES when Emulator is LibsnesCore or BsnesCore or SubBsnesCore:

```

Comment:
Don't need type checks anymore, and you can keep the label assignment as `SNESSubMenu.Text = Emulator.SystemId is VSystemID.Raw.SGB ? "&SGB" : "&SNES";`

Reply:
> you can keep the label assignment as SNESSubMenu.Text = Emulator.SystemId is VSystemID.Raw.SGB ? "&SGB" : "&SNES";

No other core is doing that and when thinking about it I also found it kind of counterintuitive that the label text changes even though the menu is exactly the same. Not really opposed to changing the text dynamically, but it doesn't seem useful to me right now.